Package.swift

```diff
@@ -41,18 +41,18 @@ let package = Package(
                 "SwiftyBeaver",
                 "Gloss",
                 "PIALibraryUtilObjC",
-                "SwiftEntryKit",
-                "PopupDialog",
+                .product(name: "PopupDialog", package: "PopupDialog", condition: .when(platforms: [.iOS])),
+                .product(name: "SwiftEntryKit", package: "SwiftEntryKit", condition: .when(platforms: [.iOS])),
                 .product(name: "Lottie", package: "lottie-ios"),
                 .product(name: "Reachability", package: "Reachability.swift"),
                 .product(name: "PIAKPI", package: "mobile-ios-releases-kpi"),
                 .product(name: "PIACSI", package: "mobile-ios-releases-csi"),
                 .product(name: "PIARegions", package: "mobile-ios-releases-regions"),
                 .product(name: "PIAAccount", package: "mobile-ios-releases-account"),
                 .product(name: "PIAWireguard", package: "mobile-ios-wireguard"),
```

Comment:
@kp-said-rehouni , we need to do the same with WG, right? We also need to only include it on iOS platform atm

Reply:
Right! Thanks for the reminder @kp-laura-sempere. 
Updating it!

Reply:
After checking the impact, it's probably better to do it on a different PR. PIAWireguard contains some dependencies that PIALibrary uses, such as Alamofire. We need to add also those dependencies to PIALibrary and add availability checks for all the classes that import PIAWireguard. What do you think @kp-laura-sempere?

Reply:
No problem Said, I can also do this update on the WG crash ticket that I'm working on
I already had to move the Alamofire dependency to PIALibrary in the branch I'm working on

Reply:
Thank you @kp-laura-sempere!